zephyr/wrapper.c

```diff
@@ -72,18 +72,32 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
```

Comment:
How about for IMX (the above #if-else statement?

Reply:
isn't it intact for the IMX case here?